Makefile

```diff
@@ -17,8 +17,11 @@ src/commcare_cloud/help_cache/ansible-playbook.txt:
 docs/commcare-cloud/commands/index.md : src/commcare_cloud/* src/commcare_cloud/*/* src/commcare_cloud/*/*/*
 	manage-commcare-cloud make-docs > docs/commcare-cloud/commands/index.md
 
+docs/changelog/index.md : src/commcare_cloud/* src/commcare_cloud/*/* src/commcare_cloud/*/*/*
```

Comment:
To the right of the `:` should go the files that this build step depends on. `docs/commcare-cloud/commands/index.md` above depends on or could possibly depend on pretty much everything under `src/commcare-cloud` (thought I probably could have been more specific about it): at least `src/commcare_cloud/commcare_cloud.py`, `src/commcare_cloud/commands/`, and `src/commcare_cloud/manage_commcare_cloud/`.

Here, I think we depend on really just `src/commcare_cloud/manage_commcare_cloud/*` (which is covered by what you put here) but then also, importantly, `docs/changelog/*-*.md`, since these are essentially the input files used in `make-changelog-index`.

Currently if I run `make`, then add another file to `docs/changelog/`, and then run `make` again, nothing is done on the second run, whereas the desired behavior would be for `docs/changelog/index.md` to get rebuilt.

Reply:
I noticed this after merging and fixed it with a few other makefile changes